http/jsonapi/response.go

```diff
@@ -317,6 +316,26 @@ func visitModelNode(model interface{}, included *map[string]*Node,
 					return nil, fmt.Errorf("decimal.MarshalJSONWithoutQuotes needs to be turned on to export decimals as numbers")
 				}
 				node.Attributes[args[1]] = json.RawMessage(d.String())
+			} else if fieldValue.Type() == reflect.TypeOf(new(decimal.Decimal)) {
```

Comment:
These changes should be covered by unit tests as well.

Reply:
Covered the changes with unit tests. Vulnerability scan is killing the CI tests again :face_with_spiral_eyes: